src/StatusQ/Controls/StatusTabButton.qml

```diff
@@ -25,17 +25,21 @@ import StatusQ.Components 0.1
 TabButton {
     id: root
 
-    property alias badge: statusBadge
+    readonly property alias badge: statusBadge
 
     leftPadding: 12
     rightPadding: 12
 
     background: Item { }
 
-    contentItem: Item {
+    contentItem: MouseArea {
         implicitWidth: contentItemGrid.implicitWidth
         implicitHeight: contentItemGrid.implicitHeight + 15
 
+        cursorShape: Qt.PointingHandCursor
+        acceptedButtons: Qt.NoButton
+        enabled: root.enabled
+
```

Comment:
Don't you need `hoverEnabled` here as well?

Reply:
No, `hoverEnabled` is not necessary for handling cursor shape. The is example in official docs for that: https://doc.qt.io/qt-6/qml-qtquick-mousearea.html#cursorShape-prop.